core/java/android/ext/settings/ClipboardConstants.java

```diff
@@ -0,0 +1,9 @@
+package android.ext.settings;
+
+/**
```

Comment:
Just to stay consistent with other constants, could this comment be one line?:

```suggestion
/** @hide */
```